tests/environment-1.yml

```diff
@@ -1,7 +1,4 @@
 name: xeus-python-kernel-1
-channels:
-  - https://repo.mamba.pm/emscripten-forge
```

Comment:
Curious why the channels can be removed here?

Looking at the code, there seems to be a distinct path when an environment file exists and when it does not:

https://github.com/jupyterlite/xeus/blob/eda12f8c2e50bd7ca4b171e38492de652a678cb4/jupyterlite_xeus/add_on.py#L130-L147